xtas/core.py

```diff
@@ -84,7 +84,7 @@ def configure(config, import_error="raise", unknown_key="raise"):
 
     app.config_from_object(config.get('CELERY', 'xtas._defaultconfig.CELERY'))
     es = config.get('ELASTICSEARCH', _defaultconfig.ELASTICSEARCH)
-    _config['ELASTICSEARCH'] = es
+    config['ELASTICSEARCH'] = es
```

Comment:
Merged too quickly: now the local config object is being overwritten. I'm on it.